meta-resin-common/recipes-support/resin-mounts/resin-mounts/mnt-boot.mount

```diff
@@ -7,7 +7,7 @@ Conflicts=umount.target
 [Mount]
 What=/dev/disk/by-label/resin-boot
 Where=/mnt/boot
-Type=vfat
+Type=ubifs
```

Comment:
No, this won't work. Ubi needs special mounting. Don't think systemd supports it.